src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Making the switch"
+tags:
+- Learning paths
+- Customer experience
+- Migration
+metaDescription: "How to fully migrate your data monitoring from Datadog to New Relic."
+---
+
+import journeyFullStack from 'images/journey_diagram_full-stack-monitoring.svg'
+
+If you've followed our tutorial series, you've learned a little about how New Relic works firsthand. You've instrumented your entities for infrastructure, applications, and logs to start really seeing how your stack works together. You've setup a premade dashboard to visualize the data across your stack, and you've set up an alert or two to help monitor issues that could arise with your systems. 
+
+<img
+    title="Full_stack"
+    alt="A diagram detailing New Relic's full stack monitoring capabilities"
+    src={journeyFullStack}
+/>
+
+But what if you need to do more with your data? What if you need to use synthetic monitors to test your agents to make sure everything runs smoothly without customer interaction? What if you need to query hyper-specific data points to uncover in-depth information that you can't find anywhere else? What if you need an observability solution that's low on cost but still high on functionality? 
+
+Digging deeper into New Relic features will help you do all that and more, and you can continue to explore the platform [for free](https://newrelic.com/signup). One of our core principle is to help you do your best work every day using their data at every phase of development. Once you make the switch, you'll never look at your data the same way again.
+
+## Do more with your data [#do-more]
+
+New Relic has a host of features that enable you to maximize the information you get from your data. Once you have your infrastructure, application, and log monitors set up, you can use any of the following observability features to help you get the most out of your stack.
+
+<CollapserGroup>
+    <Collapser
+        id="synthetics"
+        title="Synthetics"
+    >
+    Create tests using synthetic input to simulate traffic, resolve ability issues, and measure uptime with zero effects on your users. Test what you need when you need it, no matter how specific your needs are. 
+        * Simulated traffic can be public or private across thousands of possible locations.
+        * Find out about sub-optimal performance before your customers have a chance to know about them.
+        * Get full context on your stack to identify specifically where your issues are arising from.
+        * Use our synthetic testing in pre-production environments, or adjust any pages that are already in production.
+
+    Learn more about synthetics in our [short overview](https://newrelic.com/platform/synthetics), or see our [introduction to synthetics](/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring/) documentation.
```

Comment:
Identation on this is a bit odd in the build.

Reply:
Think I fixed it